amethyst_rendy/src/submodules/gather.rs

```diff
@@ -36,14 +35,13 @@ impl CameraGatherer {
         #[cfg(feature = "profiler")]
         profile_scope!("gather_cameras");
 
-        let (active_camera, cameras, transforms, dimensions) = <(
+        let (active_camera, cameras, transforms) = <(
             Read<'_, ActiveCamera>,
             ReadStorage<'_, Camera>,
             ReadStorage<'_, Transform>,
-            ReadExpect<'_, ScreenDimensions>,
         )>::fetch(res);
 
-        let defcam = Camera::standard_2d(dimensions.width(), dimensions.height());
+        let defcam = Camera::standard_2d(1.0, 1.0);
```

Comment:
I'm not sure that this change means; I'd trust @jaynus ' review on this

Reply:
It's about removing dependency on window size. In pretty much all cases, your defined camera kicks in. The default is instead made to be of size 1.0. I think we could really just not render stuff if there is no camera and display a warning instead. I don't want to introduce that yet though.

Reply:
Yup!